README.md

```diff
@@ -36,6 +37,8 @@ Community contributions are most welcome, and there are a number of ways to part
 - [Submit bugs for the MAAS website](https://github.com/canonical-web-and-design/maas.io)
 - [Contribute to MAAS documentation](https://maas.io/docs/writing-guide)
 
+When submitting a PR, please take note that MAAS UI uses the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format. To help you conform to this, you can run `yarn commit` or `git cz` instead of `git commit` for an interactive prompt.
```

Comment:
I think "or `git cz` instead" should be "or `git-cz` instead".

Reply:
I think `git cz` is for when you have the tool installed globally. Maybe this sentence can make note of that distinction.

Reply:
I might just suggest `yarn commit` to avoid confusion.